src/algorithms/tracking/AmbiguitySolver.cc

```diff
@@ -84,183 +84,62 @@ void AmbiguitySolver::init(std::shared_ptr<spdlog::logger> log) {
   m_core        = std::make_unique<Acts::GreedyAmbiguityResolution>(m_acts_cfg, logger().clone());
 }
 
-std::tuple<
-    std::unique_ptr<edm4eic::TrajectoryCollection>,
-    std::unique_ptr<edm4eic::TrackParametersCollection>, std::unique_ptr<edm4eic::TrackCollection>,
-    std::vector<ActsExamples::ConstTrackContainer*>, std::vector<ActsExamples::Trajectories*>>
+
+std::tuple<std::vector<ActsExamples::ConstTrackContainer*>, std::vector<ActsExamples::Trajectories*>>
 AmbiguitySolver::process(std::vector<const ActsExamples::ConstTrackContainer*> input_container,
                          std::vector<const ActsExamples::Trajectories*> input_traj,
                          const edm4eic::Measurement2DCollection& meas2Ds) {
```

Comment:
seems input_traj is not used anywhere?

Reply:
Yeah, I think that's right. It is not used with the updated method of making the solved trajectories. I will remove that.